gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/AbstractConnectorPluginService.java

```diff
@@ -96,6 +98,16 @@ protected ConnectorPluginEntity convert(Plugin plugin) {
         if (connectorFactory.supportedApi() != null) {
             entity.setSupportedApiType(ApiType.fromLabel(connectorFactory.supportedApi().getLabel()));
         }
+
+        if (connectorFactory instanceof EntrypointConnectorFactory) {
+            EntrypointConnectorFactory entrypointConnectorFactory = (EntrypointConnectorFactory) connectorFactory;
+            if (entrypointConnectorFactory.supportedListenerType() != null) {
```

Comment:
normally never null. As for the other fields we ignore if null I do the same here 🤷‍♂️